AutoService.BLL/Services/SaleService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AutoService.BLL.DTO;
+using AutoService.DAL.UnitOfWork;
+using AutoService.DAL.Entities;
+using AutoService.BLL.Infrastructure;
+
+namespace AutoService.BLL.Services
+{
+    public class SaleService : ISaleService
+    {
+        IUnitOfWork db;
+
+        public SaleService(IUnitOfWork uow)
```

Comment:
How this UnitOfWork pattern works?
Where transaction is started and finished? 
I don't see any usages of method Save